### PR TITLE
docs: entity-relationship diagram for cross-service linking

### PR DESCRIPTION
**Pull request by iloveagent57**
**Description:**

https://2u-internal.atlassian.net/browse/ENT-6747

### CODE REVIEW

File: docs/decisions/0004-cross-service-data-linking.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+References with enterprise-access
+=================================
+
+The ``SubsidyAccessPolicy`` model's table contains a reference to the ``uuid`` of ``LearnerCreditSubsidy``
+(or any other implementation of the abstract ``Subsidy`` model).  This is necessary for the policy to
+know which subsidy record should have the transaction for a given redemption action.
+
+In ``enterprise-subsidy``, the Ledger ``Transaction`` table will contain a ``subsidy_access_policy`` field
+that refers to the access policy record that allowed a given transaction to be created.
```
Comment (pwnage101) [inline docs/decisions/0004-cross-service-data-linking.rst:36]: I think I accidentally called it `policy_uuid` in my PR

https://github.com/openedx/openedx-ledger/pull/3/files

I think `subsidy_access_policy` is more explicit, but what about `subsidy_access_policy_uuid`?

**Review by pwnage101 — APPROVED** [review]

File: docs/decisions/0004-cross-service-data-linking.rst [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- Puts us at some small risk of introducing coupling between the data of different services.
+
+Rejected Alternatives
+*********************
+
+No serious alternatives considered - these sorts of bi-directional references became somewhat necessary
+when we adopted a design that encapsulates the bounded context of `Enterprise subsidies` as a discrete
+microservice.
+
+.. _entity-relationship diagram: https://github.com/openedx/enterprise-subsidy/blob/40aca442173338ee42ef699a9fe1d2c1895ace43/docs/decisions/0004-cross-service-data-linking.pdf
```
Comment (pwnage101) [inline docs/decisions/0004-cross-service-data-linking.rst:69]: Also, can you convert this to PNG so that it can be viewed directly in github?

Comment (iloveagent57) [reply docs/decisions/0004-cross-service-data-linking.rst:69]: ah you know, I did that, but never pushed it to the branch!

File: docs/decisions/0004-cross-service-data-linking.rst (hunk 1 above)
Comment (pwnage101) [reply docs/decisions/0004-cross-service-data-linking.rst:36]: we discussed and agreed on `subsidy_access_policy_uuid` over slack

**Pull request merged**